packages/google_maps_flutter/google_maps_flutter_android/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -937,12 +937,20 @@ public void moveCamera(@NonNull Messages.PlatformCameraUpdate cameraUpdate) {
   }
 
   @Override
-  public void animateCamera(@NonNull Messages.PlatformCameraUpdate cameraUpdate) {
+  public void animateCamera(
+      @NonNull Messages.PlatformCameraUpdate cameraUpdate, @Nullable Long durationMilliseconds) {
     if (googleMap == null) {
       throw new FlutterError(
           "GoogleMap uninitialized", "animateCamera called prior to map initialization", null);
     }
-    googleMap.animateCamera(Convert.cameraUpdateFromPigeon(cameraUpdate, density));
+    if (durationMilliseconds != null) {
+      googleMap.animateCamera(
+          Convert.cameraUpdateFromPigeon(cameraUpdate, density),
```

Comment:
Nit: pull out Convert.cameraUpdateFromPigeon... in to a shared variable.

Reply:
Done